insights/parsers/proc_keys.py

```diff
@@ -0,0 +1,91 @@
+"""
+ProcKeys - File ``/proc/keys``
+==============================
+
+This parser reads the content of ``/proc/keys``.
+
+"""
+
+from insights import Parser, parser
+from insights.parsers import SkipException, get_active_lines
+from insights.specs import Specs
+
+
+@parser(Specs.proc_keys)
+class ProcKeys(Parser):
+    """
+    Class ``ProcKeys`` parses the content of the ``/proc/keys``.
+
+    Attributes:
+        ID (string):            The ID of the key, expressed in hexadecimal.
```

Comment:
I didn't see that you set these `keys` as attributes of the parser in its code, but just keys of the dicts in the list.

I think you can remove them from the docstring if it's expected that to hold the parsing results as a list. Or illustrate them in the section for the `dict of the list` but not the `Attributes` of the parser.